packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -17,17 +17,19 @@ export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement>
   isResizable?: boolean;
   /** Callback for resize end. */
   onResize?: (width: number, id: string) => void;
-  /** The minimum size of a resizable drawer, in pixels. Defaults to the starting width of the drawer. */
+  /** The minimum size of a drawer, in pixels. */
   minSize?: number;
-  /** The maximum size of a resizable drawer, in pixels. Defaults to the max width of the parent container. */
+  /** The starting size of a resizable drawer, in pixels. */
+  defaultSize?: number;
```

Comment:
is there a requirement for this to be in pixels? The var doesn't care what the unit is - px, %, em vw, etc... and that works with the CSS that keeps it within the `minSize`/`maxSize`

Reply:
I could change it to a string property. The user would have to specify `px` or `%` in that case. Would that be okay?

Reply:
I had it in pixels so I could limit how much the `mousemove` would calculate and set a new size (throwing out the sizes outside the min/max). But since that's being done in CSS I can probably remove them.